src/main/java/transforms/CombineDateTimeTransform.java

```diff
@@ -29,18 +31,21 @@
      * @param datePattern: list of patterns of date
      * @param timeColumn:  column of time
      * @param timePattern: list of patterns of time
+     * @param timezone: name of the timezone
      */
     @JsonCreator
     public CombineDateTimeTransform(
             @JsonProperty( Constants.DATE_COLUMN ) String dateColumn,
             @JsonProperty( Constants.DATE_PATTERN ) String[] datePattern,
             @JsonProperty( Constants.TIME_COLUMN ) String timeColumn,
-            @JsonProperty( Constants.TIME_PATTERN ) String[] timePattern
+            @JsonProperty( Constants.TIME_PATTERN ) String[] timePattern,
+            @JsonProperty( Constants.TIMEZONE ) Optional<String> timezone
     ) {
         this.dateColumn = dateColumn;
         this.datePattern = datePattern;
         this.timeColumn = timeColumn;
         this.timePattern = timePattern;
+        this.timezone = TimeZone.getTimeZone( timezone.orElse("America/New_York") );
```

Comment:
If `timezone` isn't present it would be better to use `TimeZones.America_NewYork` than hardcode the string in multiple places

Reply:
like hardcode "America/New_York" as `Constants.DEFAULT_TIMEZONE`?

Or use `TimeZones.America_NewYork` in all transformations? Why is the string more hardcoded than using `TimeZones.America_NewYork`?

Reply:
I mean use `TimeZones.America_NewYork` (although you could set a DEFAULT_TIMEZONE of TimeZones.America_NewYork and use that)

Reply:
It's added to the defaults.